server/src/test/java/org/spine3/server/entity/storage/EntityQueryShould.java

```diff
@@ -45,69 +53,96 @@
     public void not_accept_nulls() {
         new NullPointerTester()
                 .setDefault(EntityIdFilter.class, EntityIdFilter.getDefaultInstance())
+                .setDefault(QueryParameters.class, QueryParameters.newBuilder()
+                                                                  .build())
                 .testStaticMethods(EntityQuery.class, NullPointerTester.Visibility.PACKAGE);
     }
 
     @Test
     public void support_equality() {
-        final Object defaultId = 0;
-        final ProjectId someId = Sample.messageOfType(ProjectId.class);
-        final ProjectId someIdCopy = ProjectId.newBuilder(someId)
-                                              .build();
-        final ProjectId otherId = Sample.messageOfType(ProjectId.class);
-        final Collection<?> idsA = Arrays.asList(someId, defaultId);
-        final Collection<?> idsB = Collections.emptyList();
-        final Collection<?> idsC = Arrays.<Object>asList(otherId, someIdCopy);
-        final Column<?> someColumn = mock(Column.class);
-        final Column<?> otherColumn = mock(Column.class);
-
-        final Object someValue = "anything";
-        final Object otherValue = 5;
-
-        final Map<Column<?>, Object> paramsA = new HashMap<>(2);
-        paramsA.put(someColumn, someValue);
-        paramsA.put(otherColumn, otherValue);
-
-        final Map<Column<?>, Object> paramsB = new HashMap<>(1);
-        paramsB.put(someColumn, otherValue);
-
-        final Map<Column<?>, Object> paramsC = new HashMap<>(1);
-        paramsA.put(otherColumn, someValue);
-
-        final Map<Column<?>, Object> paramsCCopy = new HashMap<>(1);
-        paramsA.put(otherColumn, someValue);
-
-        final EntityQuery<?> queryA = EntityQuery.of(idsA, paramsA);
-        final EntityQuery<?> queryB = EntityQuery.of(idsB, paramsB);
-        final EntityQuery<?> queryC = EntityQuery.of(idsC, paramsC);
-        final EntityQuery<?> queryD = EntityQuery.of(idsA, paramsC);
-        final EntityQuery<?> queryE = EntityQuery.of(idsB, paramsB);
-        final EntityQuery<?> queryF = EntityQuery.of(idsC, paramsCCopy);
-
-        new EqualsTester()
-                .addEqualityGroup(queryA)
-                .addEqualityGroup(queryB, queryE)
-                .addEqualityGroup(queryC, queryF)
-                .addEqualityGroup(queryD)
-                .testEquals();
+        final EqualsTester tester = new EqualsTester();
+        addEqualityGroupA(tester);
+        addEqualityGroupB(tester);
+        addEqualityGroupC(tester);
+        addEqualityGroupD(tester);
+        tester.testEquals();
+    }
+
+    private static void addEqualityGroupA(EqualsTester tester) {
```

Comment:
If it is not possible to name the methods properly, you must add comments to each of these methods, explaining what the equality group represent.

Just extracting the methods is NOT what we have discussed previously.